www/js/survey/enketo/AddedNotesList.tsx

```diff
@@ -33,7 +35,7 @@ const AddedNotesList = ({ timelineEntry, additionEntries }: Props) => {
     if (isMultiDay(beginTs, stopTs)) {
       const beginTsZoned = moment.parseZone(beginTs*1000).tz(timezone);
       const stopTsZoned = moment.parseZone(stopTs*1000).tz(timezone);
-      d = getFormattedDateAbbr(beginTsZoned.unix(), stopTsZoned.unix());
+      d = getFormattedDateAbbr(beginTsZoned.toISOString(), stopTsZoned.toISOString());
```

Comment:
Note to self: this was changed in 7eedb92ce96580238ca4444328560275ed4f84b1 because 

"--getFormattedDateAbbr actually takes an ISO string, not a unix timestamp. This was changed some time ago but not updated here (thanks typescript)"